packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { EXPLORER_URL } from '../../config';
+import { checkAccountAvailable } from '../../redux/actions/account';
+import { clearLocalAlert, showCustomAlert } from '../../redux/actions/status';
+import { selectBalance } from '../../redux/slices/account';
+import { actions as nftActions } from '../../redux/slices/nft';
+import NonFungibleTokens, { NFT_TRANSFER_GAS } from '../../services/NonFungibleTokens';
+import isMobile from '../../utils/isMobile';
+import Balance from '../common/balance/Balance';
+import FormButton from '../common/FormButton';
+import Modal from '../common/modal/Modal';
+import ModalFooter from '../common/modal/ModalFooter';
+import SafeTranslate from '../SafeTranslate';
+import ReceiverInputWithLabel from '../send/components/ReceiverInputWithLabel';
+import AvatarSuccessIcon from '../svg/AvatarSuccessIcon';
+import EstimatedFees from '../transfer/EstimatedFees';
+
+
+const StyledContainer = styled.div`
+    &&& {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 0 0 0 0;
+
+        img {
+            width: 100%  ;
+            max-width: 472px;
+            border-radius: 8px;
+            margin-bottom: 16px;
```

Comment:
```suggestion
            margin: 16px 0;
```

Reply:
is this setting the margin bottom or top?

Reply:
Let's set 16px top/bottom here as suggested